`GetLightBlock()` panics if there is no stored trusted block for a specific height

```
panic: ciphertext is too short

goroutine 15 [running]:
github.com/tendermint/tendermint/light/store/db.(*dbs).LightBlock(0x7f6b18170380, 0x24a)
	/home/ubuntu/go/pkg/mod/github.com/tendermint/tendermint@v0.34.19/light/store/db/db.go:117 +0x1cc
github.com/tendermint/tendermint/light.(*Client).TrustedLightBlock(0x7f6b18492300, 0x40?)
	/home/ubuntu/go/pkg/mod/github.com/tendermint/tendermint@v0.34.19/light/client.go:409 +0x4d
github.com/medibloc/panacea-doracle/panacea.QueryClient.GetLightBlock(...)
	/home/ubuntu/repos/panacea-doracle/panacea/query_client.go:320
github.com/medibloc/panacea-doracle/event.RegisterOracleEvent.EventHandler({{0x7f6b02577658?, 0x7f6b18034180?}}, {{0x7f6b180386a0, 0x1f}, {0x7f6b024640c0, 0x7f6b18162140}, 0x7f6b1815c300})
	/home/ubuntu/repos/panacea-doracle/event/register_oracle_event.go:62 +0x2f6
github.com/medibloc/panacea-doracle/event.(*PanaceaSubscriber).subscribe.func1({0x7f6b02576560, 0x7f6b184b21d0})
	/home/ubuntu/repos/panacea-doracle/event/subscriber.go:58 +0x117
created by github.com/medibloc/panacea-doracle/event.(*PanaceaSubscriber).subscribe
	/home/ubuntu/repos/panacea-doracle/event/subscriber.go:56 +0x27e
```

The error `panic: ciphertext is too short` occurs before the DB returned nil.

In the `GetLightBlock()`, we should use `safeVerifyLightBlockAtHeight(ctx, height)` instead of `TrustedLightBlock(height)`.

--- a/panacea/query_client.go
+++ b/panacea/query_client.go
@@ -317,7 +317,7 @@ func (q QueryClient) GetOracleRegistration(oracleAddr, uniqueID string) (*oracle
 }
 
 func (q QueryClient) GetLightBlock(height int64) (*tmtypes.LightBlock, error) {
-	return q.lightClient.TrustedLightBlock(height)
+	return q.safeVerifyLightBlockAtHeight(context.Background(), height)
 }
 
 func (q QueryClient) GetOracleParamsPublicKey() (*btcec.PublicKey, error) {

